modules/EPMC.py

```diff
@@ -58,39 +58,47 @@ def main(cooccurrenceFile, outputFile, local=False):
         # Reading file:
         spark.read.parquet(cooccurrenceFile)
 
+        # Casting integer pmid column to string:
+        .withColumn("pmid", pf.col('pmid').cast(StringType()))
+
+        # publication identifier is a pmid if available otherwise pmcid
+        .withColumn(
+            'publicationIdentifier',
+            pf.when(pf.col('pmid').isNull(), pf.col('pmcid'))
+            .otherwise(pf.col('pmid'))
+        )
+
         # Filtering for disease/target cooccurrences:
         .filter(
             (pf.col('type') == 'GP-DS') &  # Filter gene/protein - disease cooccurrence
-            (pf.col('isMapped') == True) &  # Filtering for mapped cooccurrences
-            (pf.col('pmid').isNotNull()) &  # Excluding publications without pmid
+            (pf.col('isMapped')) &  # Filtering for mapped cooccurrences
+            # Making sure at least the pmid or the pmcid is given:
+            (pf.col('pmid').isNotNull() | pf.col('pmcid').isNotNull()) &
```

Comment:
Since we define this helper column above, seems sensible to use it here for filtering as well:

```suggestion
            (pf.col('publicationIdentifier').isNotNull()) &
```

Reply:
Absolutely.